airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_40_6_001__AddGeographyColumnToConnections.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import java.util.Arrays;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.Catalog;
+import org.jooq.DSLContext;
+import org.jooq.EnumType;
+import org.jooq.Schema;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.jooq.impl.SchemaImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class V0_40_6_001__AddGeographyColumnToConnections extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_40_6_001__AddGeographyColumnToConnections.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    // Warning: please do not use any jOOQ generated code to write a migration.
+    // As database schema changes, the generated jOOQ code can be deprecated. So
+    // old migration may not compile if there is any generated code.
+    final DSLContext ctx = DSL.using(context.getConnection());
+
+    addGeographyEnumDataTypes(ctx);
+    addGeographyColumn(ctx);
+  }
+
+  private static void addGeographyEnumDataTypes(final DSLContext ctx) {
+    ctx.createType(GeographyType.NAME)
+        .asEnum(Arrays.stream(GeographyType.values()).map(GeographyType::getLiteral).toList())
+        .execute();
+  }
+
+  private static void addGeographyColumn(final DSLContext ctx) {
```

Comment:
@terencecho just realized, I think we want all existing connection record to be set to 'AUTO' during this migration, right?